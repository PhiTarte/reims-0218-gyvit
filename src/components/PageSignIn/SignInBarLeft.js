import React from "react"
import { Container } from "reactstrap"
import SignInTitle from "./SignInTitle"
import SignInForm from "./SignInForm"
import SignInGetStarted from "./SignInGetStarted"

const PageSignIn = () => (
  <Container
    style={{
      paddingTop: "290px",
      marginLeft: "10%",
      backgroundColor: "#ffffff",
      marginRight: "50px"
    }}
  >
    <SignInTitle />
    <SignInForm />
  
    <SignInGetStarted />
  </Container>
)

export default PageSignIn
