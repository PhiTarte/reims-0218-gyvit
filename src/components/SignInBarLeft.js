import React from "react"
import { Container } from "reactstrap"
import SignInButton from "./SignInButton"
import SignInTitle from "./SignInTitle"
import SignInEmail from "./SignInEmail"
import SignInGetStarted from "./SignInGetStarted"

const PageSignIn = () => (
  <Container
    style={{
      "padding-top": "290px",
      "margin-left": "60px",
      "background-color": "#ffffff",
      height: "1067px",
      width: "460px"
    }}
  >
    <SignInTitle />
    <SignInEmail />
    <SignInButton />
    <SignInGetStarted />
  </Container>
)

export default PageSignIn
