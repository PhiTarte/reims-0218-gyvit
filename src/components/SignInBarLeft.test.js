//TODO: test:  navbar top with search bar, notification icon and profile.

import React from "react"
import ReactDOM from "react-dom"
import renderer from "react-test-renderer"
import SignInBarLeft from "./SignInBarLeft"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<SignInBarLeft />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders correctly", () => {
  const tree = renderer.create(<SignInBarLeft />).toJSON()
  expect(tree).toMatchSnapshot()
})
