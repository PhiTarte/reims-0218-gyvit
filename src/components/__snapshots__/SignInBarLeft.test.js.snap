// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container"
  style={
    Object {
      "background-color": "#ffffff",
      "height": "1067px",
      "margin-left": "60px",
      "padding-top": "290px",
      "width": "460px",
    }
  }
>
  <div>
    <h1
      style={
        Object {
          "color": "#3b425d",
          "font-size": "24px",
          "font-stretch": "normal",
          "font-style": "normal",
          "font-weight": "normal",
          "height": "24px",
          "letter-spacing": "normal",
          "line-height": "normal",
          "text-align": "left",
        }
      }
    >
      Sign it to Gyvit
    </h1>
    <p
      style={
        Object {
          "color": "#9da5c9",
          "font-size": "18px",
          "font-stretch": "normal",
          "font-style": "normal",
          "font-weight": "normal",
          "height": "18px",
          "letter-spacing": "normal",
          "line-height": "normal",
          "margin-top": "15px",
          "text-align": "left",
          "width": "200px",
        }
      }
    >
      Enter your details below
       
    </p>
  </div>
  <form
    className=""
    style={
      Object {
        "margin-bottom": "40px",
        "margin-top": "60px",
      }
    }
  >
    <div
      className="form-group"
    >
      <label
        className=""
        htmlFor="email"
      >
        Email 
      </label>
      <input
        className="form-control"
        id="IdEmail"
        name="email"
        placeholder="Email"
        type="email"
      />
    </div>
    <div
      className="form-group"
    >
      <label
        className=""
        htmlFor="password"
        style={
          Object {
            "margin-top": "30px",
          }
        }
      >
        Password
      </label>
      <a
        className="nav-link"
        href="#"
        onClick={[Function]}
        style={
          Object {
            "color": "#7a57d1",
            "display": "inline",
            "font-size": "14px",
            "font-stretch": "normal",
            "font-style": "normal",
            "font-weight": "normal",
            "letter-spacing": "normal",
            "line-height": "normal",
            "padding-left": "110px",
            "text-align": "right",
          }
        }
      >
        Forgot password?
      </a>
      <input
        className="form-control"
        id="IdPassword"
        name="password"
        placeholder="Password"
        type="password"
      />
    </div>
  </form>
  <div>
    <button
      className="btn btn-info"
      onClick={[Function]}
      style={
        Object {
          "background-image": "linear-gradient(to left, #57aad1, #7a57d1)",
          "border-radius": "100px",
          "height": "54px",
          "width": "192px",
        }
      }
      type={undefined}
    >
      Sign in
    </button>
  </div>
  <div
    style={
      Object {
        "margin-top": "120px",
      }
    }
  >
    <p>
      Don't have an account ?
      <a
        className="nav-link"
        href="#"
        onClick={[Function]}
        style={
          Object {
            "display": "inline",
          }
        }
      >
        Get started
      </a>
    </p>
  </div>
</div>
`;
